src/GmailConnection.php

```diff
@@ -168,7 +168,13 @@ public function setBothAccessToken( $token )
 	public function saveAccessToken( array $config )
 	{
 		$fileName = $this->getFileName();
-		$file = "gmail/tokens/$fileName.json";
+		$user = auth()->user();
```

Comment:
If you are going to use only the id, you could use `auth()->id()`.

Reply:
I think we cannot assume directly that the user is logged in to laravel?